rust/rsc/src/rsc/main.rs

```diff
@@ -306,7 +303,7 @@ fn launch_blob_eviction(
             tracing::info!(%eligible, "Blobs eligible for eviction");
```

Comment:
This message is technically incorrect with the limit in place. This could be "At least N blobs eligible for eviction"